src/main.rs

```diff
@@ -271,10 +272,17 @@ async fn reindex(
         after.remove("order");
       }
 
+      let ctx_str: Vec<&str> = ctx.iter().map(|s| s.as_str()).collect();
+
       // delete batch from document if it exists
-      after.remove("batch");
+      match ctx_str[..] {
```

Comment:
Comment out this, because it's here to fix old bugs of writing calculated branches